.github/workflows/sub-ci-integration-tests-gcp.yml

```diff
@@ -344,18 +344,17 @@ jobs:
   # Otherwise, if the state rebuild was skipped, runs immediately after the build job.
   lightwalletd-update-sync:
     name: lightwalletd tip update
-    needs: [lightwalletd-full-sync, get-available-disks]
```

Comment:
As agreed, let's remove this changes from this file and let's leave it for a different PR